src/lib/api.js

```diff
@@ -1,6 +1,47 @@
 import axios from 'axios';
 
 const api = {
+    responseInterceptorId: null,
+    requestInterceptorId: null,
+    /**
+     * Adds a function to Axios that intercepts a response
+
```

Comment:
extra line?

Reply:
Not sure why this doesn't show as outdated.